nxdrive/engine/engine.py

```diff
@@ -411,6 +411,10 @@ def _save_remote_parent_infos(
         self.dao.update_config("dt_last_remote_location_ref", remote_ref)
         self.dao.update_config("dt_last_duplicates_behavior", duplicate_behavior)
 
+    def _save_local_selected_folder(self, path: Path) -> None:
+        """Store the last selected folder in Direct Transfer."""
+        self.dao.update_config("dt_last_local_selected_location", path.parent)
```

Comment:
Move the line to the method above. You could also rename it, as it will be outdated.